apps/designer/app/designer/features/style-panel/shared/configs.ts

```diff
@@ -4,9 +4,30 @@ import {
   keywordValues,
   type Category,
   type StyleProperty,
+  type AppliesTo,
 } from "@webstudio-is/react-sdk";
-import type { StyleConfig, Control } from "./types";
-import { humanizeString } from "../string-utils";
+import { humanizeString } from "~/shared/string-utils";
+
+type BaseStyleConfig = {
+  label: string;
+  property: StyleProperty;
+  appliesTo: AppliesTo;
+};
+
+export type Control =
+  | "Spacing"
+  | "Combobox"
+  | "Color"
+  | "IconButtonWithMenu"
+  | "Select"
+  | "Empty";
+
+type StyleConfigWithItems = BaseStyleConfig & {
```

Comment:
hmm, why do we need an alias? seems like it can be 

```suggestion
export type StyleConfig = BaseStyleConfig & {
```